apps/drec-api/src/pods/organization/dto/new-organization.dto.ts

```diff
@@ -44,40 +44,47 @@ export class NewOrganizationDTO
   @IsString()
   vatNumber: string;
 
-  @ApiProperty({ type: String })
+  @ApiPropertyOptional({ type: String })
+  @IsOptional()
```

Comment:
many optional types here are not reflected in typescript type

Reply:
Not sure that I understand what you mean. I added the optional type to the optional fields '?'.  Can you give me an example?